### PR TITLE
PMM-3694 Add agregation of errors.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-3694

Depends on https://github.com/percona/go-mysql/pull/44

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -15,4 +15,5 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/go-mysql"
-  branch = "PMM-2.0"
+  # branch = "PMM-2.0"
+  branch = "PMM-3694-errors"
```
Comment (AlekSi) [inline Gopkg.toml]: It was merged.

Comment (askomorokhov) [reply Gopkg.toml]: Updated.

File: exporters/slow_log.go [hunk 2]
```diff
@@ -432,3 +434,11 @@ func listsToMap(k, v []string) map[string]string {
...
 	return m
 }
+
+func errListsToMap(k, v []uint64) map[uint64]uint64 {
+	m := map[uint64]uint64{}
+	for i, e := range k {
+		m[e] = v[i]
+	}
+	return m
+}
```
Comment (AlekSi) [inline exporters/slow_log.go]: I think we should move that tool to pmm-agent repository soon – it already has a different version of slow log file parser, for example.

**Comment by AlekSi** [comment]
Squash/merge when ready.

File: exporters/slow_log.go (hunk 2 above)
Comment (askomorokhov) [reply exporters/slow_log.go]: Sure, I'll do.

**Pull request merged**